themes/nekonight/nekonight-base.sh

```diff
@@ -0,0 +1,35 @@
+function _omb_theme_nekonight_git_prompt_info() {
+  local branch_name
+  branch_name=$(git symbolic-ref --short HEAD 2>&-)
+  local git_status=""
+
+  local icon_emoji="${_omb_theme_nekonight_icon_emoji:-🐱}"
+
+  if [[ -n $branch_name ]]; then
+    git_status="${_omb_prompt_bold_white} (${icon_emoji} $branch_name $(_omb_theme_nekonight_scm_git_status))${_omb_prompt_normal}"
+  fi
+
+  echo -n "$git_status"
+}
+
+function _omb_theme_nekonight_scm_git_status() {
+  local git_status=""
+
+  if git rev-list --count --left-right @{upstream}...HEAD 2>&- | grep -Eq '^[0-9]+\s[0-9]+$'; then
```

Comment:
I haven't carefully looked at this regular expression in the previous PR, but `\s` is not POSIX. It can be used in GNU grep, but it may not be the case in other `grep` implementations.

```suggestion
  if git rev-list --count --left-right @{upstream}...HEAD 2>&- | grep -Eq '^[0-9]+[[:blank:]][0-9]+$'; then
```